packages/block-editor/src/components/font-sizes/utils.js

```diff
@@ -62,6 +62,15 @@ export function getFontSizeClass( fontSizeSlug ) {
 		return;
 	}
 
+	// In the past, we used lodash's kebabCase to process slugs.
+	// By doing so, this method also accepted and converted non string values
+	// into strings. Some plugins relied on this behavior.
+	if ( 'string' !== typeof fontSizeSlug ) {
```

Comment:
I wonder if we also have to add an allowed list of characters as we are doing for color in https://github.com/WordPress/gutenberg/pull/32722/files#diff-d879f383eaa52d106b2745f1efce82798380acfad517514bb3e9ae01b4d0ed40R67 

The issue is the same: lodash's kebabCase also did that, so there may be some third-parties that relied on it.

Reply:
Pushed at https://github.com/WordPress/gutenberg/pull/32720/commits/5955ee6b78928cdc0024bb8ca00562c7f0dd6815